docs/IcepeakProtocolMultiSub.md

```diff
@@ -0,0 +1,177 @@
+# The Icepeak Multiple-Subcription-Single-Connection Protocol
+
+This protocol was developed atop the existing protocol to allow the client to subscribe to multiple "paths"/"topics" using a single websocket connection.
+
+Whereas the original existing protocol required a new websocket connection for each path the client wanted to subscribe to.
+
+# Initial Connection
+
+A client can connect to a multiple-subscription protocol connection with a URL such as:
+`ws://yourdomain.com/?method=reusable`
+
+The URL path:
+- points to the root of your icepeak service
+- has the query parameter `method` set to `reusable`
+
+The request should contain:
+- a token of autherisation about the right to establish a connection with the server **UNDER-SPECIFIED** 
+
```

Comment:
Ah? That's not the case in this implementation. 
Shall I put?:
```
The authorisation method follows the same protocol as the old protocol.
```

Reply:
Rather the implementation should be changed to not require a token for the initial connection, and we need the timeout in case no follow-up subscription comes in.